src/lib/resolver.js

```diff
@@ -15,53 +15,21 @@ var PDI = '\u2069';
 const meta = new WeakMap();
 const resolutionChain = new WeakSet();
 
-export function createEntry(node, lang) {
-  var keys = Object.keys(node);
+export function createEntry(id, entity, lang) {
+  meta.set(entity, {id, lang});
 
-  // the most common scenario: a simple string with no arguments
-  if (typeof node.$v === 'string' && keys.length === 2) {
-    return node.$v;
+  if (entity.attrs === undefined) {
```

Comment:
You might not need this check since `for...in` gracefully doesn't do anything for `undefined`.